rpa-builder/6.3/modules/ROOT/pages/toolbox-variable-handling-generate-and-modify-date-and-time.adoc

```diff
@@ -6,80 +6,126 @@ templates or arrange the available time components as required, e.g.
 day, month, and hour. You can specify the time stamp as a specific time
 in the future or past, or use the current system time. It is also
 possible to incorporate a time offset in days, hours, minutes, and
-seconds. The time stamp generated by _Generate date and time_ will be
-made available in several variables, which can be further processed in
+seconds. _Generate date and time_ creates a timestamp that is available in several variables, which you can further process in
 other _Action Steps_.
 
 == Properties
 
-*Generate Date and Time Settings*
-
-The values specified for the following fields (Day, Hour, Minute, and
-Second) will be incorporated into the calculation of the selected time
-stamp (time offset). Positive and negative numbers can be used. The
+* *Generate Date and Time Settings*
++
+_Generate date and time_ incorporates the specified values into the calculation of the selected time
+stamp (time offset). This action step supports positive and negative numbers. The
 default value is 0 in each case.
-
-* *Day* Number of days
-* **Generate day and time mode**Use the list box to select whether you
-want the _Current date and time_ or a _Custom date and time_ to be used.
-The time offsets from the Day, Hour, Minute, and Second fields will be
-applied in both cases.
-* *Hour* Number of hours
-* *Minute* Number of minutes
-* *Second* Number of seconds
-
-Other Properties that are the same in all Action Steps are described in
-the section on *Common Properties*. For reasons of
-clarity they are not shown here.
+* *Calendar week setting* 
++
+Specify either Monday or Sunday as the starting day for the week.
+* *Days to Add* 
++
+Number of days to add.
+* *Generate day and time mode*
++
+Select whether to use the *Current system date and time*, *Custom date and time*, or *Generated date and time from given variables*. The time offsets from the *Years to add*, *Months to add*, *Weeks to add*, *Days to add*, *Hours to add*, *Minutes to add*, *Seconds to add* fields apply for the calculation in all cases.
+* *Hours to add* 
++
+Number of hours to add.
+* *Minutes to add* 
++
+Number of minutes to add.
+* *Months to add* 
++
+Number of months to add.
+* *Output language culture (for the day of the week)* 
++
+Specify the output language to use for the days of the week. 
+* *Override activated*
++
+Enables overriding workday settings. 
+* *Override to day* 
++
+Specify the day to override with. 
+* *Override to first or last* 
++
+Specify to use either the first or last day of the given period (for example, last day of the month).
+* *Seconds to add* 
++
+Number of seconds to add.
+* *Start workday of week*
++
+Specify the starting working day of the week.
+* *Weeks to add* 
++
+Number of weeks to add.
+* *Workdays setting* 
++
+Select the length of the workweek between *5 days workweek* and *6 days workweek*.

```

Comment:
Is this in the UI like this? It would normally be five-day workweek, six-day workweek.